docs/src/components/Anchor.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+// eslint-disable-next-line import/no-unresolved
+import useBrokenLinks from "@docusaurus/useBrokenLinks";
```

Comment:
This looks great. Do you know why the module is reported as unresolved?

Reply:
No I don't but I see we've done this elsewhere in this folder so I didn't interrogate it at the time.